face.go

```diff
@@ -115,21 +134,21 @@ func (rec *Recognizer) recognizeFile(imgPath string, maxFaces int) (face []Face,
 	if err != nil {
 		return
 	}
-	return rec.recognize(imgData, maxFaces)
+	return rec.recognize(type_, imgData, maxFaces)
 }
 
 // Recognize returns all faces found on the provided image, sorted from
 // left to right. Empty list is returned if there are no faces, error is
 // returned if there was some error while decoding/processing image.
 // Only JPEG format is currently supported. Thread-safe.
-func (rec *Recognizer) Recognize(imgData []byte) (faces []Face, err error) {
-	return rec.recognize(imgData, 0)
+func (rec *Recognizer) Recognize(type_ int, imgData []byte) (faces []Face, err error) {
```

Comment:
We could make new function e.g. `RecognizeCNN` to avoid breaking API

Reply:
Fixed